src/components/input-field/input-field.tsx

```diff
@@ -217,6 +217,12 @@ export class InputField {
     @Prop({ reflect: true })
     public showLink = false;
 
+    /**
+     * The user's locale that determines how numbers are formatted.
+     */
+    @Prop({ reflect: true })
+    public locale: string = navigator.language;
```

Comment:
We could use global config here (`import config from '../../global/config';`)

```suggestion
    public locale: string = config.defaultLocale;
```

The icon component is an example that uses global config, which is provided using `limel-config`

Reply:
Would it be better to initialize `defaultLocale` to `en`? 

```
export class Config {
    public iconPath = '';
    public defaultLocale: string
    public featureSwitches: any = getFeatureSwitches(localStorage);
}

Reply:
no, `navigator.language` is better I think, as it's the user's preferred language